src/main/java/org/trigon/hardware/phoenix6/Phoenix6Inputs.java

```diff
@@ -30,15 +35,29 @@ public void toLog(LogTable table) {
         latestTable = table;
     }
 
+    /**
+     * Registers a threaded signal.
+     * Threaded signals use threading to process certain signals separately, keeping them running smoothly in the background.
+     *
+     * @param statusSignal         the signal
+     * @param updateFrequencyHertz the update frequency in hertz
+     */
     public void registerThreadedSignal(BaseStatusSignal statusSignal, double updateFrequencyHertz) {
         if (statusSignal == null || RobotHardwareStats.isReplay())
             return;
 
         registerSignal(statusSignal, updateFrequencyHertz);
-        statusSignal.setUpdateFrequency(50);
+        if (RobotHardwareStats.isSimulation())
+            statusSignal.setUpdateFrequency(50);
         signalToThreadedQueue.put(statusSignal.getName() + "_Threaded", signalThread.registerSignal(statusSignal));
     }
 
+    /**
+     * Registers a signal.
+     *
+     * @param statusSignal         the signal
+     * @param updateFrequencyHertz the update frequency in hertz
+     */
```

Comment:
add @ override, no need for duped jdoc

Reply:
override what? There isn't a registerSignal in BaseInputs